loader.qml

```diff
@@ -1,57 +1,193 @@
 import QtQuick 2.15
 import QtGraphicalEffects 1.12
 import QtQuick.Layouts 1.2
-import QtQuick.Controls 1.4
+import QtQuick.Controls 2.15
 import QtQuick.Dialogs 1.1
 
 ApplicationWindow {
     id: root
+    visible: true
+    minimumWidth: 640
+    minimumHeight: mysize + header.height // + menuBar.height
     property bool displayAmbient: false
     property bool round: true
+    property int mysize: 640
     property var testface: Qt.application.arguments[1]
+    readonly property var mouseWheelScale: 1/15
 
-    toolBar: ToolBar {
-        RowLayout {
-            anchors.fill: parent
-            anchors.margins: spacing
-            CheckBox {
-                id: roundCheckBox
-                text: "Round"
-                Component.onCompleted: checked = round
-                onCheckedChanged: round = checked; 
-            }
-            CheckBox {
-                id: checkBox12h
-                text: "12h time"
-                Component.onCompleted: checked = use12H.value
-                onCheckedChanged: use12H.value = checked; 
-            }
-            CheckBox {
-                id: ambientCheckBox
-                text: "Dark"
-                Component.onCompleted: checked = displayAmbient
-                onCheckedChanged: displayAmbient = checked; 
+        /*
+    menuBar: MenuBar {
+        Menu {
+            title: qsTr("&Help")
+            Action { text: qsTr("&About") }
+        }
+    }
+        */
+    header: ToolBar {
+        background: Rectangle {
+            color: "lightblue"
+        }
+        ColumnLayout {
+            RowLayout {
+                Layout.alignment: Qt.AlignCenter
+                CheckBox {
+                    id: roundCheckBox
+                    text: "Round"
+                    Component.onCompleted: checked = round
+                    onCheckedChanged: round = checked; 
```

Comment:
It would be clearer to bind "round" to roundCheckBox.checked instead ? That's the whole power of a declarative language like QML where you just say what you want instead of how you want to do it. Same for the other checkboxes. It takes a bit of time to get used to it coming from an imperative paradigm world but it's fairly nice :)

Reply:
Yes, *much* cleaner as you suggest.  I can't remember why I wrote it like this!